### PR TITLE
fix the fp8 topk_config.correction_bias is none bug

### PR DESCRIPTION
**Pull request by rainj-me**
## Motivation

Fix bug when the topk_config.correction_bias is None and trigger none object .to(dtype) ops.

## Modifications

- fp8.py conditional error

## Checklist

- [x] Format your code according to the [Format code with pre-commit](https://docs.sglang.ai/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [x] Add unit tests according to the [Run and add unit tests](https://docs.sglang.ai/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [x] Update documentation according to [Write documentations](https://docs.sglang.ai/developer_guide/contribution_guide.html#write-documentations).
- [x] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.ai/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.ai/developer_guide/contribution_guide.html#benchmark-the-speed).

### CODE REVIEW

File: python/sglang/srt/layers/quantization/fp8.py [hunk 1]
```diff
@@ -1108,7 +1108,7 @@ def apply_with_router_logits(
             and topk_config.topk_group is not None
         ), "Current trtllm_fp8_block_scale_moe kernel does not support these two arguments as None"
 
-        if topk_config.correction_bias is None:
+        if topk_config.correction_bias is not None:
             correction_bias = topk_config.correction_bias.to(x.dtype)
         else:
             correction_bias = None
```
[the remark this answers is not in the document]
Comment (rainj-me) [reply python/sglang/srt/layers/quantization/fp8.py]: refactored

**Review by Fridge003 — APPROVED** [review]
LGTM

**Comment by rainj-me** [comment]
> LGTM

I can't merge this PR, not sure why.

**Comment by rainj-me** [comment]
> LGTM

I can't merge this PR, not sure why.

**Comment by Fridge003** [comment]
> > LGTM
> 
> I can't merge this PR, not sure why.

@zhyncs

**Review by mickqian — APPROVED** [review]

**Review by ch-wan — APPROVED** [review]
